web/pdf_viewer.css

```diff
@@ -37,6 +37,15 @@
   }
 }
 
+body.wait::before {
+  content: "";
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  z-index: 100000;
+  cursor: wait;
+}
+
```

Comment:
Considering where the JS code is added (outside of e.g. `PDFViewer`), in this particular case it probably makes more sense to place this rule in `web/viewer.css` instead; e.g. in https://github.com/mozilla/pdf.js/blob/47791a4c8088582d3f3374beab7e2af4d35153fd/web/viewer.css#L223-L226